doc/Command_Index_User.tex

```diff
@@ -1717,6 +1693,65 @@ \subsubsection*{Bar distances}
 Same as \texttt{maxbaroffsettextleft@eol} but when the center of the text goes right of the center of the bar.
 \end{gdimension}
 
+\subsubsection{Distances tied to font size}
+In some cases it is desirable to link a distance with the size of the text font, rather than the score's staff size.  This can be accomplished in the following way:
+
+\begin{enumerate}
+  \item The distance should be defined in terms of the various \texttt{\textbackslash fontdimen} values (there are 7).  These should be prepended by \verb=\the= and can be combined or scaled using \verb=\dimexpr= and \verb=\glueexpr= (which also need to be prepended by \verb=\the= when used).
```

Comment:
maybe explaining that these are values defined by the font themselves and accessed by TeX, not defined by TeX, and are mostly related to space